src/atasks.cc

```diff
@@ -180,13 +180,14 @@ void TaskBarApp::paint(Graphics &g, const YRect &/*r*/) {
     }
 
     ref<YIcon> icon(getFrame()->getIcon());
+    bool drew_icon = false;
```

Comment:
What about a config option which enables the alternative behavior? I, for one, am not really happy with the icon-less taskbar entries.

Reply:
Actually, 
{code}
    bool drew_icon = false;
{code}
should probably just be
{code}
    bool drew_icon;
{code}
because it gets initialsed later, and doesn't matter if it's true or false in that declaration.

Reply:
I added a new commit, did you get notifified of that, and does that resolve it for you?